README.md

```diff
@@ -138,6 +146,36 @@ packager URL directly, first add a Chrome extension to send an
 `AMP-Cache-Transform: any` request header. Otherwise, follow the above
 "Demonstrate privacy-preserving prefetch" instructions.
 
+##### Security Considerations
+
+Signed exchanges come with some [security
+considerations](https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html#security-considerations)
+that publishers should consider. A starting list of recommendations based on
+that:
+
+ * Use different keys for the signed exchange cert and the TLS cert.
+ * Only sign public content that's OK to be shared with crawlers.
+ * Don't sign personalized content. (It's OK to sign content that includes
+   static JS that adds personalization at runtime.)
+ * Be careful when signing inline JS; if it includes a vulnerability, it may be
+   possible for attackers to exploit it without intercepting the network path,
+   for up to 7 days.
+
+#### Testing productionization without a valid certificate
```

Comment:
This section is not yet possible in the release branch (requires `-invalidcert` support from #278)

Reply:
Ugh, thanks. Pushed this in too much of a hurry. I'll send you a fix. Guess there will be a conflict at release-time after all. :/